docs/5___ssd.adoc

```diff
@@ -102,7 +103,7 @@ The following XML attributes are specified for the Connector element:
 |===
 |Attribute |Description
 |name a|
-This attribute gives the connector a name, which *SHALL* be unique within the given model element, and, for components, *MUST* match the name of a relevant variable/port in the underlying component implementation. In the case of referenced FMUs this *MUST* match the name of the relevant variable in the referenced FMU.
+This attribute gives the connector a name, which *SHALL* be unique within the given model element, and, for components, *MUST* match the name of a relevant variable/port in the underlying component implementation. In the case of referenced FMUs this *MUST* match the name of the relevant variable or alias in the referenced FMU.
```

Comment:
Not specific to this commit but the "Use [one sentence per line](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line)" rule is violated here.

Reply:
Yes we should fix occurrences of this in some cleanup PR.